### PR TITLE
 fix(locales): fix typos in French translation

### PR DESCRIPTION
**Pull request by Fl0wyn**

### CODE REVIEW

File: src/locales/fr.json.ts [hunk 1]
```diff
@@ -44,12 +44,12 @@ exportAsMessages('_locales/fr/messages.json', {
   options_importBlacklistDialog_title: 'Import',
   options_importBlacklistDialog_fromFile: 'Importer depuis un fichier',
   options_importBlacklistDialog_selectFile: 'Sélectionner un fichier',
-  options_importBlacklistDialog_fromPB: 'Importer depuis une Blocklist personnelle',
+  options_importBlacklistDialog_fromPB: 'Importer depuis une Blacklist personnelle',
```
Comment (Elliot67) [inline src/locales/fr.json.ts:47]: There is a confusing mix between Blacklist & Blocklist in the en translations :
```
  // The label for the select to import from Personal Blocklist.
  options_importBlacklistDialog_fromPB: 'Import from Personal Blocklist',
```
Don't know which one should be used.

File: src/locales/fr.json.ts [hunk 2]
```diff
@@ -87,7 +87,7 @@ exportAsMessages('_locales/fr/messages.json', {
   options_turnOnSyncDialog_title: 'Activer la synchronisation',
   options_turnOnSyncDialog_turnOnSyncButton: 'Activer',
   options_turnOnSyncDialog_altFlowDescription:
-    "Vous serez peut être demandé d'accepter les permissions pour accéder à $1 avant l'authentification, mais vous données personnelles ne seront PAS enregistrées dans ce domaine.",
+    "Il se peut que l'on vous demande les permission pour accéder à $1 avant l'authentification, avant l'authentification, mais vos informations personnelles ne seront PAS enregistrées dans ce domaine.",
```
Comment (Elliot67) [inline src/locales/fr.json.ts:90]: - [ ] `les permission` -> `les permissions`
- [ ] ` $1 avant l'authentification, avant l'authentification, mais` -> ` $1 avant l'authentification, mais`

**Review by Elliot67 — CHANGES_REQUESTED** [review]

**Comment by iorate** [comment]
I don't understand French, so I can't judge which tranlastaion is better.
If @Fl0wyn and @Elliot67 reach an agreement (@Elliot67 approve the changes), I will merge this PR.

File: src/locales/fr.json.ts (hunk 1 above)
Comment (iorate) [reply src/locales/fr.json.ts:47]: "Personal Blocklist" is a proper noun, so I think it is better to leave it as it is.

**Comment by Fl0wyn** [comment]
I think it's ok with the new changes from @Elliot67
We corrected each other

**Comment by iorate** [comment]
@Fl0wyn OK, could you please commit the changes suggested by @Elliot67 ?

**Comment by iorate** [comment]
Merged. Thank you for your contribution

**Pull request merged**